### PR TITLE
Modified section of Power Flow that was rebranded as Power Automated…

### PR DESCRIPTION
**Pull request by edwintorres**
Modified section of Power Flow that was rebranded as Power Automated recently.

### CODE REVIEW

**Comment by edwintorres** [comment]
I'm running the following command on my computer, and it's running well. Not sure why it's failing here

` pa11y-ci --sitemap https://engineering.18f.gov/sitemap.xml --sitemap-find https://engineering.18f.gov --sitemap-replace http://localhost:4000 --sitemap-exclude "/*.pdf"`

**Comment by edwintorres** [comment]
It seems related to this issue: https://github.com/pa11y/pa11y/issues/570

File: _pages/sharepoint.md [hunk 1]
```diff
@@ -135,14 +135,15 @@ connected to other live data sources. The disadvantage is that Power BI is
...
-needed.
+### Power Automate
+
+Another one of Microsoft's "Power" tools, Power Automate, is not a display tool, but it is a workflow automation system. If we need 
+* to move data between systems, 
+* send notifications or messages when certain things happen on SharePoint or with various documents,
+* connect to or create connectors to APIs,
+* approval processes, 
+
+then we can configure a Flow to do those actions. This may not be a large part of a SharePoint site, but it could be an important capability if needed.
```
Comment (neilmb) [inline _pages/sharepoint.md:146]: **question (non-blocking):** With the name change to "Automate", are these actions still called "Flows"?

**Review by neilmb — APPROVED** [review]
Thanks for the update @edwintorres.

File: _pages/sharepoint.md (hunk 1 above)
Comment (edwintorres) [reply _pages/sharepoint.md:146]: Yes, according to this page in [MS Docs](https://docs.microsoft.com/en-us/power-automate/flow-types).

**Pull request merged**